src/css/toolbox-layers-list.css

```diff
@@ -53,13 +53,14 @@
  */
 
 .layers-list {
+  display: table;
```

Comment:
This breaks the scroll if there are too many layers. 

Do we really need to change the layout from flex to table? What was the issue with the current flex approach?

Reply:
I had some problems with the alpha (opacity) symbol to stay in place when starting the renaming. So I thought a table would be easier to manage. But I didn't notice that I broke the scroll, good catch ;) . I now use the flex again, let me know what you think, as I'm not so proficient with css.